src/components/CardEditor/CardEditForm/CardEditFormItems/ImageCardFormItems/ImageCardFormSettings.jsx

```diff
@@ -0,0 +1,211 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  RadioButtonGroup,
+  RadioButton,
+  FormGroup,
+  ToggleSmall,
+} from 'carbon-components-react';
+import { gray10, gray80, white } from '@carbon/colors';
+
+import { settings } from '../../../../../constants/Settings';
+import ColorDropdown from '../../../../ColorDropdown/ColorDropdown';
+
+const { iotPrefix } = settings;
+
+const propTypes = {
+  /** card data value */
+  cardConfig: PropTypes.shape({
+    id: PropTypes.string,
+    title: PropTypes.string,
+    size: PropTypes.string,
+    type: PropTypes.string,
+    content: PropTypes.shape({
+      hideMinimap: PropTypes.bool,
+      hideHotspots: PropTypes.bool,
+      hideZoomControls: PropTypes.bool,
+      id: PropTypes.string,
+      displayOption: PropTypes.string,
+      background: PropTypes.string,
+    }),
+    interval: PropTypes.string,
+    showLegend: PropTypes.bool,
+  }),
+  /** Callback function when form data changes */
+  onChange: PropTypes.func.isRequired,
+  i18n: PropTypes.shape({
+    displayOptions: PropTypes.string,
+    colorTitleText: PropTypes.string,
+    hideMap: PropTypes.string,
+    hideZoom: PropTypes.string,
+    zoomLevel: PropTypes.string,
+  }),
+};
+
+const defaultProps = {
+  cardConfig: {},
+  i18n: {
+    displayOptions: 'Image display options',
+    colorTitleText: 'Background color',
+    hideMap: 'Hide mini map',
+    hideZoom: 'Hide zoom controls',
+    zoomLevel: 'Max zoom level',
+  },
+};
+
+export const backGroundColor = (hex) => {
```

Comment:
But on second thought, I don't think we need this function, and it would be nice not have to duplicate the color definitions. If we create a constant for these values currently defined directly on the ColorDropdown prop:

```
const colors = [
 { carbonColor: gray10, name: 'gray10' },
 { carbonColor: gray80, name: 'gray80' },
 { carbonColor: white, name: 'white' },
]
```
And later when we set selected color we can do:

`selectedColor={ colors.find(color => color.carbonColor === cardConfig.content?.background)}`

Reply:
Good call.